### PR TITLE
feat: add link to binding wallet when checkout (DBIO-404)

### PR DESCRIPTION
**Pull request by ciniiia**
### JIRA Link
https://blocksphere2020.atlassian.net/browse/DBIO-404?atlOrigin=eyJpIjoiNzk0NWE3Y2I2NjZkNDMyY2JmYzQ5NDI5NWQwODA2MzAiLCJwIjoiaiJ9

### Changelog / Description
- add button that will directing user to binding wallet when checkout, if wallet has not been connected yet

### CODE REVIEW

File: src/views/Dashboard/Customer/RequestTestCheckout/index.vue [hunk 1]
```diff
@@ -124,9 +124,14 @@
       :imgPath="alertImgPath"
       :imgWidth="imgWidth"
       @toggle="dialogAlert = $event"
-      @click="bindingWallet()"
       @close="actionAlert()"
     ></DialogAlert>
+    <WalletBinding
+      :show="showWalletBinding"
+      @toggle="showWalletBinding = $event"
+      @status-wallet="({ status, img }) => connectWalletResult(status, img)"
```
Comment (bukandicki) [inline src/views/Dashboard/Customer/RequestTestCheckout/index.vue:132]: LGTM mbak @ciniiia 👍🏻 
Cuman bagusnya ini di buat method khususnya aja , hindarin inline method kaya gini mbak @ciniiia biasanya sering ada unexpected error

**Review by bukandicki — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules
 /dist
+package-lock-json
```
Comment (kevinjanada) [inline .gitignore:4]: package-lock-json ini setauku sebaiknya jgn di gitignore ya. mnurut kalian gmn?
@ciniiia @bukandicki @luthfiazhari

Comment (bukandicki) [reply .gitignore:4]: yep setuju mas, supaya kepantau terus mas 👍🏻

Comment (kevinjanada) [reply .gitignore:4]: aku request change ya mba @ciniiia

**Review by kevinjanada — CHANGES_REQUESTED** [review]
minta tolong package-lock-json nya tidak usah di ignore ya mba

File: .gitignore (hunk 2 above)
Comment (ciniiia) [reply .gitignore:4]: okayy siap mas mas 👍🏻

**Review by kevinjanada — APPROVED** [review]

**Pull request merged**